sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -124,19 +131,24 @@
     private final ClientLogger logger = new ClientLogger(ServiceBusAdministrationAsyncClient.class);
     private final ServiceBusManagementSerializer serializer;
     private final RulesImpl rulesClient;
+    private final TokenCredential tokenCredential;
 
     /**
      * Creates a new instance with the given management client and serializer.
      *
      * @param managementClient Client to make management calls.
      * @param serializer Serializer to deserialize ATOM XML responses.
+     * @param credential Credential to get additional tokens if necessary
+     *
+     * @throws NullPointerException if any one of {@code managementClient, serializer, credential} is null.
      */
     ServiceBusAdministrationAsyncClient(ServiceBusManagementClientImpl managementClient,
-        ServiceBusManagementSerializer serializer) {
+        ServiceBusManagementSerializer serializer, TokenCredential credential) throws NullPointerException {
```

Comment:
We only add throws for checked exceptions.

Reply:
Ah this was a modification I did based on https://github.com/Azure/azure-sdk-for-java/pull/25146#discussion_r741296267
Will remove